advanced/dapps/react-dapp-v2/src/contexts/ClientContext.tsx

```diff
@@ -320,6 +317,22 @@ export function ClientContextProvider({
     origin,
   ]);
 
+  useEffect(() => {
+    const claimedOrigin =
+      localStorage.getItem("wallet_connect_dapp_origin") || origin;
+    let interval: NodeJS.Timer;
+    // simulates `UNKNOWN` validation by removing the verify iframe thus pereventing POST message
```

Comment:
```suggestion
    // simulates `UNKNOWN` validation by removing the verify iframe thus preventing POST message
```

Reply:
Could we elaborate on why an interval is needed here for context. You mentioned in the PR description but not intuitive here, e.g. because we try to create an iframe on .com and .org